docs/01-getting-started/02-project-structure.mdx

```diff
@@ -91,7 +91,7 @@ This page provides an overview of the file and folder structure of a Next.js pro
 | --------------------------------------------------------------------------------------------------------------- | ----------------------------------- | ------------------------ |
 | [`favicon`](/docs/app/api-reference/file-conventions/metadata/app-icons#favicon)                                | `.ico`                              | Favicon file             |
 | [`icon`](/docs/app/api-reference/file-conventions/metadata/app-icons#icon)                                      | `.png` `.svg`                       | App Icon file            |
```

Comment:
```suggestion
| [`icon`](/docs/app/api-reference/file-conventions/metadata/app-icons#icon)                                      |  `.ico` `.jpg`  `.jpeg` `.png` `.svg`                       | App Icon file            |
```

Thanks @bigyanse! It looks like the one above for the static `icon` is missing some files, let's add these. `icon.ico` might look weird, but it's supported :)